content/python/concepts/regex/terms/metacharacters/metacharacters.md

```diff
@@ -0,0 +1,98 @@
+---
+Title: 'Metacharacters'
+Description: 'Matches any digit (0-9) in a given pattern.'
```

Comment:
I think this should be similar to line 16 since you're discussing all of these characters, not just those that match digits

Reply:
Haha meant to change that. Will do.